seacrowd/sea_datasets/ara_close/ara_close.py

```diff
@@ -0,0 +1,222 @@
+    or degree of language relatedness can improve ARA in a low-resource setting. We collect short stories written in three \
+    languages in the Philippines{---}Tagalog, Bikol, and Cebuano{---}to train readability assessment models and explore the \
+    interaction of data and features in various cross-lingual setups. Our results show that the inclusion of CrossNGO, \
+    a novel specialized feature exploiting n-gram overlap applied to languages with high mutual intelligibility, \
+    significantly improves the performance of ARA models compared to the use of off-the-shelf large multilingual \
+    language models alone. Consequently, when both linguistic representations are combined, we achieve state-of-the-art \
+    results for Tagalog and Cebuano, and baseline scores for ARA in Bikol.",
+}
+"""
+
+_DATASETNAME = "ara_close"
+
+_DESCRIPTION = """\
+The dataset contribution of this study is a compilation of short fictional stories \
+written in Bikol for readability assessment. The data was combined other collected \
+Philippine language corpora, such as Tagalog and Cebuano. The data from these languages \
+are all distributed across the Philippine elementary system's first three grade \
+levels (L1, L2, L3). We sourced this dataset from Let's Read Asia (LRA), Bloom Library, \
+Department of Education, and Adarna House. \
+"""
+
+_HOMEPAGE = "https://github.com/imperialite/ara-close-lang"
+
+_LANGUAGES = ["bcl", "ceb"]  # We follow ISO639-3 language code (https://iso639-3.sil.org/code_tables/639/data)
+
+_LICENSE = Licenses.CC_BY_4_0.value  # example: Licenses.MIT.value, Licenses.CC_BY_NC_SA_4_0.value, Licenses.UNLICENSE.value, Licenses.UNKNOWN.value
+
+_LOCAL = False
+
+_URLS = {
+    "bcl": "https://raw.githubusercontent.com/imperialite/ara-close-lang/main/data/bikol/bik_all_data.txt",
+    # 'tgl': '', # file for tgl language was deleted
+    "ceb": "https://raw.githubusercontent.com/imperialite/ara-close-lang/main/data/cebuano/ceb_all_data.txt",
+}
+
+_SUPPORTED_TASKS = [Tasks.READABILITY_ASSESSMENT]
+
+_SOURCE_VERSION = "1.0.0"
+
+_SEACROWD_VERSION = "1.0.0"
+
+
+class AraClose(datasets.GeneratorBasedBuilder):
+    f"""{_DESCRIPTION}"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    BUILDER_CONFIGS = (
+        [
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_source",
+                version=datasets.Version(_SOURCE_VERSION),
+                description=f"{_DATASETNAME} source schema",
+                schema="source",
+                subset_id=f"{_DATASETNAME}",
+            ),
+            SEACrowdConfig(
+                name=f"{_DATASETNAME}_seacrowd_text",
+                version=datasets.Version(_SEACROWD_VERSION),
+                description=f"{_DATASETNAME} SEACrowd schema",
+                schema="seacrowd_text",
+                subset_id=f"{_DATASETNAME}",
+            ),
```

Comment:
Loading all languages in a single config is now optional. I think it's safe to remove this and just focus on the two language subsets.